index.xhtml

```diff
@@ -708,5 +716,32 @@
       </li>
     </ul>
   </section>
+
+  <section id="rules">
+    <h2>Rules</h2>
+    <p>The discord server, the subreddit and the ##programminglanguages
+    IRC channel follow these rules:</p>
+    <ol>
+      <li>Don't be an asshole:
```

Comment:
Both vague and inflammatory.

Reply:
I'm not a fan of the "Don't be an asshole" language too, but is this really the right place for commenting/critiquing the respective CoCs? Discord already has these posted in the #rules-and-info channel.

Reply:
I don't think this should even be posted on the website, which represents many affiliated but distinct communities, each which may have their own policies. These policies should only be listed in the respective communities, not here.

So, on further thought, this section should be removed.

Reply:
The subreddit and discord both use the same moderation policy. Only with changes made to make it specific to each, eg posts on reddit or messages on discord. The desire here is to have the same for the IRC channel #programminglanguages. After I opened this PR I was reminded that the website actually _belongs_ to #proglangdesign and isn't as "shared" as I'd assumed. And I'm on the fence about all of this.

And yes, no "rules" posted on the website should have any impact on any languages there, I don't think it reads like that but that is definitely something to consider.

Reply:
I agree with @brendanzab This wasn't my favourite wording, but it was favoured by the majority of people moderating the discord server and the subreddit. The wording should be a different issue.

Reply:
I think we're in favor of sharing the website, but putting a policy on the front page implies that it applies to all listed forums/channels, and there isn't a good reason to list it there anyway.

Furthermore, using offensive language could give a bad impression to newcomers, and I'd like the website to be inviting, not a warning sign to keep people away.

If you'd like to create another page specific to the subset where it applies, then I'd be fine with that, but I think it should be reworded to avoid unintentional bias e.g. some cultures/beliefs/etc may be less comfortable than others with such language. The best way to avoid such biases is to keep rules as objective as possible, for example "stay on topic" is fairly objective.